packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -923,6 +924,18 @@ class MenuItemButton extends StatefulWidget {
   /// {@endtemplate}
   final bool closeOnActivate;
 
+  /// The direction in which the menu item expands.
+  ///
+  /// If the menu item button is created by the [MenuAnchor] or [MenuBar], then
+  /// this property is ignored.
+  ///
+  /// If [menuFlexDirection] is [Axis.vertical], the menu will be expanded vertically.
+  /// If [menuFlexDirection] is [Axis.horizontal], then the menu  will be
+  /// expanded horizontally.
+  ///
+  /// Defaults to [Axis.vertical].
+  final Axis menuFlexDirection;
```

Comment:
How about `overflowDirection` or `overflowAxis` as a name?  This one isn't great: it has menu, which is redundant, and Flex which might indicate a `Flex` widget or just being "flexible", which this isn't.